cfgov/cal/views.py

```diff
@@ -10,17 +10,19 @@
 from rest_framework import generics
 
 from django.shortcuts import render, render_to_response
-from django.http import HttpResponse
+from django.http import HttpResponse, HttpResponseRedirect
 from django import forms
 from django.utils.safestring import mark_safe
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/2263/commits/8b1241190c12d6ca8aef3094ad43dcefe3459d7e.